modules/quantcastBidAdapter.js

```diff
@@ -25,29 +25,61 @@ export const storage = getStorageManager(QUANTCAST_VENDOR_ID, BIDDER_CODE);
 
 function makeVideoImp(bid) {
   const video = {};
-  if (bid.params.video) {
-    video['mimes'] = bid.params.video.mimes;
-    video['minduration'] = bid.params.video.minduration;
-    video['maxduration'] = bid.params.video.maxduration;
-    video['protocols'] = bid.params.video.protocols;
-    video['startdelay'] = bid.params.video.startdelay;
-    video['linearity'] = bid.params.video.linearity;
-    video['battr'] = bid.params.video.battr;
-    video['maxbitrate'] = bid.params.video.maxbitrate;
-    video['playbackmethod'] = bid.params.video.playbackmethod;
-    video['delivery'] = bid.params.video.delivery;
-    video['placement'] = bid.params.video.placement;
-    video['api'] = bid.params.video.api;
-  }
```

Comment:
why not keep this block so either place populates it? I think this is why your tests are failing